### PR TITLE
Trans classfc

### PR DESCRIPTION
**Pull request by JackWang0318**
DONE:

Transformed all .md files into .ipynb files
Deleted all .md files
Deleted seealso part in 'introduction-to-classfication.ipynb'
TO DO:

Plz review and tell me if anything needs to be modified.

### CODE REVIEW

File: open-machine-learning-jupyter-book/ml-fundamentals/classification/introduction-to-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,1455 @@
+{
```
Comment (bestfw) [inline open-machine-learning-jupyter-book/ml-fundamentals/classification/introduction-to-classification.ipynb:1]: <p>Please delete the warning output separately.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: open-machine-learning-jupyter-book/ml-fundamentals/classification/more-classifiers.ipynb [hunk 2]
```diff
@@ -0,0 +1,924 @@
+{
```
Comment (bestfw) [inline open-machine-learning-jupyter-book/ml-fundamentals/classification/more-classifiers.ipynb:1]: <p>Same</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: open-machine-learning-jupyter-book/ml-fundamentals/classification/yet-other-classifiers.ipynb [hunk 3]
```diff
@@ -0,0 +1,521 @@
+{
```
Comment (bestfw) [inline open-machine-learning-jupyter-book/ml-fundamentals/classification/yet-other-classifiers.ipynb:1]: <p>Same</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by bestfw** [comment]
And what is the csv you add.

**Comment by JackWang0318** [comment]
> And what is the csv you add.

sorry, What do you mean? Can you give a specific reference (in which file and where)?

**Comment by bestfw** [comment]
> > And what is the csv you add.
> 
> sorry, What do you mean? Can you give a specific reference (in which file and where)?

image
This csv, what's this, is it a new csv file?

**Comment by bestfw** [comment]
image

**Comment by JackWang0318** [comment]
> > > And what is the csv you add.
> > 
> > 
> > sorry, What do you mean? Can you give a specific reference (in which file and where)?
> 
> image This csv, what's this, is it a new csv file?

image
 是在open-ml-book 文件夹里面的这个吗？(应该不是这个吧) 好像这个是5月份就merge了的？

**Comment by JackWang0318** [comment]
> > > And what is the csv you add.
> > 
> > 
> > sorry, What do you mean? Can you give a specific reference (in which file and where)?
> 
> image This csv, what's this, is it a new csv file?

It's not a new file, maybe I accidentally modified it, but now I've remodified it.
But now why does the PR Build book fail ? I can't understand it. I've done nothing wrong so far.

**Pull request merged**